lib/psci/psci_suspend.c

```diff
@@ -180,7 +188,8 @@ void psci_cpu_suspend_start(entry_point_info_t *ep,
 	 * Plat. management: Allow the platform to perform the
 	 * necessary actions to turn off this cpu e.g. set the
 	 * platform defined mailbox with the psci entrypoint,
-	 * program the power controller etc.
+	 * program the power controller etc. This handler executes
+	 * with data cache disabled.
 	 */
 	psci_plat_pm_ops->pwr_domain_suspend(state_info);
```

Comment:
I think this function is no longer mandatory in the presence of `pwr_domain_suspend_early()`. The presence of 1 or both is OK. So:

*   This call should be wrapped in a `if (psci_plat_pm_ops->pwr_domain_suspend)`
*   There should be assert that one of both of these function are present before they are used (further up this function).